packages/core/src/common/object.spec.ts

```diff
@@ -0,0 +1,103 @@
+// *****************************************************************************
+// Copyright (C) 2022 Ericsson and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { expect } from 'chai';
+import { deepFreeze, notEmpty, isEmpty, deepClone } from './objects';
+
+describe.only('Object', () => {
```

Comment:
The `.only` should be removed:

```suggestion
describe('Object', () => {
```

The [`.only`](https://mochajs.org/#exclusive-tests) is only useful for development purposes as it makes mocha (our suite) only execute this file. This isn't what we want when running our CI in the framework as it will skip other unit-tests.